docs/astro/sql/operators/load_file.rst

```diff
@@ -186,14 +186,26 @@ Patterns in File path
 Load file can also resolve patterns in file path, there are three types of patterns supported by load file based on the :ref:`file_location`
 
 #. **Local** - On local we support glob pattern - https://docs.python.org/3/library/glob.html
-#. **S3** - prefix in file path - https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html
-#. **GCS** - prefix and wildcard in file path - https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames
 
     .. literalinclude:: ../../../../example_dags/example_load_file.py
        :language: python
        :start-after: [START load_file_example_10]
        :end-before: [END load_file_example_10]
 
+#. **S3** - prefix in file path - https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html
```

Comment:
what about adding href instead of a plain link? I think this will look better in UI `SOME HEADING <[link URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html)>`_

Reply:
Thanks, @pankajastro. I have updated the document.